app/Http/Controllers/LanguageController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Session;
+
+class LanguageController extends Controller
+{
+    public function changeLanguage(Request $request, $language) {
+        if (!in_array($language, ['en', 'vi'])) {
```

Comment:
```suggestion
        if (!in_array($language, config('app.languages'))) {
```
bỏ các hằng số vào config